### PR TITLE
03-BUNSPACE2024 :: Convert Tab Icon To Peach

### PR DESCRIPTION
**Pull request by GrpFruit**
Add peachy img; Add img as fav icon

### CODE REVIEW

File: ui/src/pages/Home/Profile/Profile.tsx [hunk 1]
```diff
@@ -5,10 +5,10 @@ export function Profile() {
...
       <div>
-      <img className="profile__image" src="bunnyW.jpg" alt="bunnie"/>
+        <img className="profile__image" src="bunnyW.jpg" alt="bunnie"/>
       </div>
       <div className="profile__text">
-      {text}{text}
+        {text}{text}
       </div>
     </div>
   )
```
Comment (jmorales8) [inline ui/src/pages/Home/Profile/Profile.tsx:14]: revert this change as it is not part of the ticket goal

Comment (GrpFruit) [reply ui/src/pages/Home/Profile/Profile.tsx:14]: no

Comment (GrpFruit) [reply ui/src/pages/Home/Profile/Profile.tsx:14]: I updated the ticket to include minor changes.

Comment (jmorales8) [reply ui/src/pages/Home/Profile/Profile.tsx:14]: AS BECAUSE of the ticket specifications now, this is an ok change

**Comment by jmorales8** [comment]
+1
